pages/docsprint/guides/participation.md

```diff
@@ -4,4 +4,37 @@ permalink: /docsprint/guides/participation/
 
 # Doc Sprint Participation Guide
 
-Here will be a guide to participating in a doc sprint.
+Taking our documentation to the next level is critical for scaling 18F. Here are some helpful materials and resources for participating in a doc sprint.
+
+## You Will Need
+To participate in an 18F doc sprint, you will need:
+
+* A [GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/) with [two-factor authentication enabled](https://help.github.com/articles/about-two-factor-authentication/)
+
+## Ways to Contribute
```

Comment:
We need another section to highlight the "big three" bits of docs:

- READMEs, especially making sure that anyone can easily build/run a project in minutes (if not seconds)
- Onboarding guides a la https://hub.18f.us/private/18f-site/onboarding/
- Updating our [18F Guides](http://18f.github.io/guides/)

I also really like the more granular advice you give in this section. However you can think to marry the two would be wonderful.